megaparse/core/parser/entity.py

```diff
@@ -0,0 +1,35 @@
+from enum import Enum
+from typing import List, Optional
+
+
+class TagEnum(str, Enum):
+    """Possible tags for the elements in the file"""
+
+    TABLE = "TABLE"
+    TOC = "TOC"
+    HEADER = "HEADER"
+    IMAGE = "IMAGE"
+
+
+class SupportedModel(Enum):
+    GPT_4O = ("gpt-4o", None)
```

Comment:
Thats Nice ! 

Additionnal typeing can be used here  (either NamedTuples or typped dict ...)